src/js/post-911-gib-status/utils/helpers.jsx

```diff
@@ -103,32 +103,6 @@ export function notQualifiedWarning() {
   );
 }
 
-export function systemDownMessage() {
```

Comment:
`master` didn't have these. I don't know if that's right or not, but I stuck with removing them.

Again, not sure who to ping on this. @bshyong or @U-DON?